resources/linting.json

```diff
@@ -52,6 +52,20 @@
             },
             "test": "yaml-lint --version",
             "tags": ["linting"]
+        },
+        {
+            "name": "twig-linter",
+            "summary": "Standalone cli twig 3.X linter",
+            "website": "https://github.com/sserbin/twig-linter",
+            "command": {
+                "composer-bin-plugin": {
+                    "package": "sserbin/twig-linter:@dev",
+                    "namespace": "tools",
```

Comment:
To avoid conflicts with other tools, can we change `tools` to `twig-linter` please?

Reply:
@jakzal done